examples/crm/src/dashboard/Dashboard.tsx

```diff
@@ -19,10 +20,10 @@ export const Dashboard = () => (
             <LatestNotes />
         </Grid>
         <Grid item xs={12} md={3}>
-            <HotContacts />
+            <TasksList />
         </Grid>
         <Grid item xs={12} md={3}>
-            <DealsPipeline />
```

Comment:
I removed the DealsPipeline from the home page as there is already a chart for that. I kept the component for when we offer a way to customize the home page.